### DISCUSSION TITLE
YouTube “not a bot” verification

### DISCUSSION THREAD

**Discussion by Treavous** [RandomNinjaAtk/arr-scripts#298] [Q&A] [answered]
![IMG_3826](https://github.com/user-attachments/assets/968432e2-1ea7-4938-83fd-e55ba292678d)

i am getting this error. I’m running it in an Ubuntu VM docker with no display. How am I supposed to log in to verify I’m not a bot? 
Thanks for any advice

**Comment by RandomNinjaAtk** [comment, marked as answer]
You have to use yt-dlp cookies.txt feature

You place the file in the /config folder

**Comment by Treavous** [comment]
Thanks for the response. I extracted my cookies file from youtube and changed the name to "cookies.txt" and placed it in my /config folder. I have checked that it is netscape format and has the correct first line and such as per the yt-dlp guide. I am still getting the error after multiple restarts of my container. I have looked through a good chunk of different program files that contain coding or config info, but I can't seem to find where to place the --cookies option. Could you guide me to the file I should add this to? Based on your comment, I assumed I didn't need to add the --cookies option, but I figured I'd look anyways. 
Unless there is something else I'm not doing right. Also would like to confirm I don't have to have a paid youtube account do I?

**Reply by RandomNinjaAtk** [reply]
If you look in the log file, it will tell you if it found your cookies file. Check the log file to make sure it says it found it. It’s usually near the start of the log file

**Reply by Treavous** [reply]
I don't really know why it didn't work. I set up deezer for a free trial and used that. When I canceled my trial and started using the YT-DLP again it just worked 🤷‍♂️
